src/lib/datepicker/datepicker.ts

```diff
@@ -22,16 +23,19 @@ import {MdDialogRef} from '../dialog/dialog-ref';
 import {PositionStrategy} from '../core/overlay/position/position-strategy';
 import {
   OriginConnectionPosition,
-  OverlayConnectionPosition
-} from '../core/overlay/position/connected-position';
+  OverlayConnectionPosition,
+  RepositionScrollStrategy,
+  BlockScrollStrategy,
```

Comment:
looks like `BlockScrollStrategy` isn't needed, also it seems like the reposition isn't working? (if I give the last input on the datepicker demo a huge top margin, it still seems to block scroll)